client/blocks/reader-feed-item/index.tsx

```diff
@@ -171,12 +214,11 @@ export default function ReaderFeedItem( props: ReaderFeedItemProps ): JSX.Elemen
 	const SubscribeButton = (): JSX.Element => (
 		<Button
 			variant="primary"
-			disabled={ subscribeDisabled }
-			isBusy={ isSubscribing }
-			onClick={ onSubscribeClick }
+			isBusy={ isSubscribing || isUnsubscribing }
+			onClick={ onSubscribeToggle }
 			__next40pxDefaultSize
 		>
-			{ hasSubscribed ? translate( 'Subscribed' ) : translate( 'Subscribe' ) }
+			{ feed?.subscription_id ? translate( 'Unsubscribe' ) : translate( 'Subscribe' ) }
```

Comment:
This is the second time here I have seen `feed?.subscription_id` as the sole condition and want to ask "what about `site`" or the blogId version of subscription?

If the subscription status is always around this feed value, maybe we should defined `isSubscribed = feed?.subscription_id` high in the component with a comment as to why site isnt necessary, and use `isSubscribed` in place of these `feed?.subscription_id` calls.

Reply:
As described above, as per my information feed subscriptions are same for wpcom and non-wpcom feeds. Let me know if that's not the case.

Add subscription status as a const above in [415b662](https://github.com/Automattic/wp-calypso/pull/101042/commits/415b6626c5b5044b07859be07e69871cf5f633f1).